gradle.properties

```diff
@@ -1,5 +1,5 @@
 VERSION_MAJOR=2.9
-VERSION_MINOR=x
+VERSION_MINOR=1.dbvis
```

Comment:
Hey, committing changes to this file is not necessary, you can invoke the build with `gradle build -PVERSION_MINOR=1.dbvis` (which is the way we build and upload releases), so I'll ignore this when merging =)